test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -218,13 +222,24 @@ public void testInitOfUrlMetadataListerDocIdIsUrlTrue() throws Exception {
     // Suppress column name validation.
     moreEntries.put("db.everyDocIdSql", "");
     moreEntries.put("db.singleDocContentSql", "");
-    final Config config = createStandardConfig(moreEntries);
+    getObjectUnderTest(moreEntries);
+  }
+
+  /**
+   * Returns a Database adaptor instance with the supplied config overrides.
+   * Adaptor.initConfig() and Adaptor.init() have already been called.
+   */
+  private DatabaseAdaptor getObjectUnderTest(
```

Comment:
Can we put these someplace logical, like before the first test, or at least before the first test that uses a DatabaseAdaptor instance?